guides/source/layout.html.erb

```diff
@@ -32,7 +32,7 @@
         <li class="more-info"><a href="http://weblog.rubyonrails.org/">Blog</a></li>
         <li class="more-info"><a href="http://guides.rubyonrails.org/">Guides</a></li>
         <li class="more-info"><a href="http://api.rubyonrails.org/">API</a></li>
-        <li class="more-info"><a href="http://stackoverflow.com/questions/tagged/ruby-on-rails">Ask for help</a></li>
+        <li class="more-info"><a href="https://stackoverflow.com/questions/tagged/ruby-on-rails">Ask for help</a></li>
```

Comment:
Actually this file is a template, not a doc. But the template is used for generating http://guides.rubyonrails.org/.
So I'm going to merge it this time.